docs/contrib_components/api/installation.rst

```diff
@@ -0,0 +1,29 @@
+Installing Wagtail API
+======================
+
+The ``wagtailapi`` module can be installed with ``pip``:
```

Comment:
The code now lives in `wagtail/contrib/wagtailapi`. I assume this is no longer needed?